src/Accelerators/NNPA/Dialect/ZHigh/ZHighHelper.cpp

```diff
@@ -154,5 +155,21 @@ ZTensorEncodingAttr::DataLayout getZTensorLayout(Type type) {
   return ZTensorEncodingAttr::DataLayout::UNDEFINED;
 }
 
+Value getMinusBcastConst(
+    mlir::OpBuilder &builder, Location loc, FloatAttr floatAttr, Value X) {
+  ShapedType xType = X.getType().cast<ShapedType>();
+  assert(xType.hasStaticShape() && "expected static shape");
+  Type resType =
+      RankedTensorType::get(xType.getShape(), xType.getElementType());
+  int nElements = ShapedType::getNumElements(xType.getShape());
+  float val = floatAttr.getValueAsDouble() * -1.0;
+  std::vector<float> array(nElements, val);
+  DenseElementsAttr denseAttr =
+      DenseElementsAttr::get(resType, llvm::makeArrayRef(array));
```

Comment:
No need an array of N same elements, DenseElementsAttr supports `DenseElementsAttr denseAttr = DenseElementsAttr::get(resType, val);`

Reply:
Fixed. Thank you.